app/index.js

```diff
@@ -168,14 +169,16 @@ Generator.prototype.gitIsTheShit = function() {
 			if (err) me.logger.error(err);
 
 			me.logger.verbose('Git init complete');
-			git.add('.', function(err) {
-				if (err) me.logger.error(err);
-			}).commit('Initial Commit', function(err, d) {
-				if (err) me.logger.error(err);
+			done();
+			//git.add('.', function(err) {
+				//if (err) me.logger.error(err);
+
+			//}).commit('Initial WP Commit', function(err, d) {
+				//if (err) me.logger.error(err);
```

Comment:
That's what I'll do here, go ahead and make an initial commit with just that gitignore, as that's more in line with v1.4.x